src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/models/Constants.java

```diff
@@ -0,0 +1,112 @@
+package com.azure.tools.apiview.processor.analysers.models;
+
+public class Constants {
+
+    public static final String APIVIEW_JSON_SCHEMA = "https://raw.githubusercontent.com/Azure/azure-sdk-tools/00aacf7f4e224e008702c3e77ecde46d25434e44/tools/apiview/parsers/apiview-treestyle-parser-schema/CodeFile.json";
+
+    /************************************************************************************
+     * Mode Flags
+     ************************************************************************************/
+
+    /** If true, we will output two files - a standard .json file, as well as a gzipped .json.tgz file. */
+    public static final boolean GZIP_OUTPUT = false;
```

Comment:
@praveenkuttappan completely removed the compressed file support. So maybe you don't need this anymore.